config.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+// Config holds the step inputs
+type Config struct {
+	TestName       string `env:"test_name,required"`
+	BasePath       string `env:"base_path,required"`
+	Pattern        string `env:"pattern,required"`
```

Comment:
not really talkative "pattern" what about `search_pattern` or maybe something more talkative. dont forget to modify it in the syep.yml also